app/assets/stylesheets/components/autocomplete.scss

```diff
@@ -8,7 +8,8 @@
   }
 
   &__popover {
-    max-width: 300px;
+    min-width: 250px;
+    max-width: 250px;
```

Comment:
Isn't that equal to `width: 250px;`?

Reply:
It is, it is, but I ended up using this to avoid having to use `!important` as the reach UI styles override `width`.

Maybe though it's better to be explicit about that, as otherwise everyone who looks at this will wonder the same 😆 
I'll update, using !important and just add a comment as to why 👍